docs/docs/zh/contribute/join/become-a-committer.md

```diff
@@ -1,12 +1,159 @@
-# 如何成为 DolphinScheduler Committer
+# 如何成为Apache Dolphinscheduler项目的committer
 
-每个人都可以成为Apache项目的贡献者。作为一个贡献者只是意味着你对项目感兴趣并以某种方式做出贡献，从提出合理的问题（这些问题记录了项目并向开发人员提供反馈）到提供新的特性作为补丁。
+Dolphinscheduler 项目管理委员会（PMC）会评估候选人的贡献，并在合适的时间提名候选者。
 
-如果你成为对一个项目有价值的贡献者，你有可能被邀请成为一个committer。committer是ASF（Apache软件基金会）中用来表示提交特定项目的人的术语。它给你带来对项目仓库和资源写的权限。
+像许多Apache项目一样，Dolphinscheduler欢迎所有形式的贡献，包括代码贡献、博客布道、新用户指南、公开演讲，以及以各种方式宣传项目和优化项目。
 
-在Dolphinscheduler社区，如果一个committer获得大量的优秀成绩，就可以被邀请加入项目管理委员会（PMC）。
+成为committer的第一步是要学习如何为Dolphinscheduler贡献并开始为Dolphinscheduler贡献，任何人可以向项目提交补丁、使用文档、测试用例等。
 
-当您不熟悉ASF使用的开源的开发过程时，有时难以理解的一点，就是我们更重视社区而不是代码。一个强大而健康的社区将受到尊重，成为一个有趣和有益的地方。更重要的是，一个多元化和健康的社区
-可以长时间的持续支持代码，即使个别公司在这个领域来来往往，也是如此。
+PMC会定期基于活跃贡献者们对Dolphinscheduler的贡献从他们中提名增加新的committer，要被提名为committer的条件如下：
 
-更多详细信息可以在[这里](https://community.apache.org/contributors/)找到
+1. 对Dolphinscheduler有持续的贡献：候选人应该对Dolphinscheduler有重大贡献，在整个项目中做出广泛的贡献，并且至少贡献了一个主要的组件，并在其中扮演了 "owner"的角色。
+2. 贡献的质量：候选人提交代码的质量应该比其他贡献者更高，此外他们应该表现出足够的专业知识来审查每个pr，包括确保它们符合Dolphinscheduler的工程实践（可测试性，文档，API稳定性，代码风格，等等）。委员会与committer对Dolphinscheduler的软件质量和可维护性共同负责。请注意，对Dolphinscheduler关键部分的贡献，比如它的核心模块，在评估质量时将会有更高的标准，这些领域的贡献者将面临更多的修改审查。
+3. 社区参与：候选人在所有的社区互动中应该有一个建设性的和友好的态度，他们还应该在开发和用户列表中活跃，帮助指导新的贡献者和用户。在设计讨论中，候选人应该保持专业和开发的态度，即使面对分歧时也要有合理的沟通。
+
+## 提名新的committer
+
+在Dolphinscheduler中，committer提名只能由现有的PMC成员开始。如果一个新的提交者觉得他/她有资格，他/她应该联系任何现有的PMC成员并进行讨论。如果这一点在PMC的一些成员中得到了认同，那么这个过程就会启动。
+
+建议采取以下步骤（仅需要现有2个PMC成员就可以发起）：
+1. 发送一封标题为"[讨论]晋升xxx为新的committer"的电子邮件到`private@dolphinscheduler.apache.org`，并在邮件中列出该候选人的重要贡献，这样就可以提名流程。
+2. 保持讨论要超过3天，但不超过1周，除非有任何明确的反对。
+3. 如果PMC普遍同意该提议，请发送一封题为"[投票]提名xxx成为新的committer"的电子邮件到`private@dolphinscheduler.apache.org`
+4. 保持投票过程超过3天，但不超过1周。如果有三张+1票，且+1票>-1票，则视为 "共识批准"。注：+1票 > -1票。
```

Comment:
A positive result is at least 3 + 1 votes with **NO** vetoes, see: https://community.apache.org/newcommitter.html

Reply:
Resolved.